core/src/main/scala/org/apache/spark/deploy/client/AppClient.scala

```diff
@@ -99,8 +98,10 @@ private[spark] class AppClient(
               return
             }
             logInfo("Connecting to master " + masterAddress.toSparkURL + "...")
-            val masterRef =
-              rpcEnv.setupEndpointRef(Master.SYSTEM_NAME, masterAddress, Master.ENDPOINT_NAME)
+            val masterRef = rpcEnv.setupEndpointRef(
```

Comment:
I see changes like this in lots of places. These are not arguments with default values, so calling them like this is completely unnecessary and just adds noise. Can you please revert changes like this from your patch?